src/client/index.js

```diff
@@ -34,4 +35,19 @@ module.exports.YotiClient = class YotiClient {
   performAmlCheck(amlProfile) {
     return amlService.performAmlCheck(amlProfile, this.pem, this.applicationId);
   }
+
+  /**
+   * @desc given a dynamic sharing request get a custom qr code denoted by the
+   * policy stated in the request.
+   * @param {*} dynamicSharingRequest is the request that contains the policy.
+   * @returns {Promise} constaining a dynamic
+   */
+  getDynamicPolicyQRCodeLink(dynamicPolicyRequest) {
```

Comment:
this can be simply `getQRCodeLink` as `getDynamicAttributeListQRCodeLink` would be ridiculous :-D